sites/postcss-preset-env/src/static/js/features/index.js

```diff
@@ -5,6 +5,11 @@
 		return;
 	}
 
+	const hashElements = [];
```

Comment:
`hashElements` was not yet defined on the first call to `onParsed`

![Uploading Screenshot 2022-05-23 at 23.48.06.png…]()